File: AirlineBookingBackend/imports/methods/carrier.js
import { Mongo } from 'meteor/mongo';

export const Carriers = new Mongo.Collection('carriers');

Meteor.methods({
  'carrier.addCarrier'(carrierName) {
      var carrierDetails = {
        carrierName: carrierName,
        planes: []
      }

      Carriers.insert(carrierDetails)
    }
})

File: AirlineBookingBackendPython/app/BookingApp/routes/carrier.py
from BookingApp import app
from flask import request
from BookingApp.utils import response_utils as rp
from BookingApp.utils import db_utils as dbu
import json


@app.route('/addCarrier', methods=['POST'])
def add_carrier():
    data = json.loads(request.data)
    
    if 'carrierName' in data:
    	dbu.insert_one({
    		'carrierName': data['carrierName'],
    		'planeCount': 0,
    		'planes': []
    	}, 'carriers')


    return rp.create_response({})

@app.route('/getCarriers')
def get_carriers():
    data = dbu.find({}, 'carriers')

    return rp.create_response(data)